### DISCUSSION TITLE
Instead of openai can we use different local llm's through litellm ?

### DISCUSSION THREAD

**Discussion by Greatz08** [Codium-ai/cover-agent#246] [General]
> OPENAI_API_KEY set in your environment variables, which is required for calling the OpenAI API.

If you can give option to set different api base then we can set selfhost and use litellm api as base which is openai API compatible and then we can use litellm key as openai key to connect and use 100+ LLM models easily so if it possible then do update the readme information and if not then please make some changes to make this possible so that many more llm's can be used easily without any issue.

**Comment by EmbeddedDevops1** [comment]
Did you see this section in the README? https://github.com/qodo-ai/qodo-cover?tab=readme-ov-file#using-other-llms
